packages/devtools/lib/src/memory/memory_protocol.dart

```diff
@@ -60,6 +60,7 @@ class MemoryTracker {
     }
 
     final VM vm = await service.getVM();
+    // TODO(terry): Need to handle a possible Sentinel being returned.
     final List<Isolate> isolates =
```

Comment:
are you getting sentinels for the isolates? That is a bit strange unless the isolate has shut down or something.

Reply:
I did see it once - I think?  Look at trying to repro or add temporary logic to catch it if it happens again.